Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM debian:jessie
```

Comment:
Sure, sounds good.  Does the vagrant file do the same thing as the docker and ansible?